knime_extension/src/nodes/spatialstatistics.py

```diff
@@ -59,90 +60,125 @@ class spatialWeights:
     This node constructs a contiguity spatial weights matrix from the input data.
     """
 
-    geo_col = knut.geo_col_parameter(description="Geometry column.")
+    geo_col = knut.geo_col_parameter(
+        description="The name of the geometry column in the input data."
+    )
+
+    id_col = knext.ColumnParameter(
```

Comment:
Load parameter backward compatible. For an example see here: https://github.com/spatial-data-lab/knime-geospatial-extension/blob/main/knime_extension/src/nodes/spatialtool.py#L601

Reply:
Code changed please look here: https://github.com/spatial-data-lab/knime-geospatial-extension/blob/main/knime_extension/src/nodes/spatialtool.py#L664